### PR TITLE
added Negar-Nasiri to before I die

### PR DESCRIPTION
**Pull request by negar-75**

### CODE REVIEW

File: CONTRIBUTORS.md
Comment (XanderRubio) [inline CONTRIBUTORS.md:1]: Great work, @negar-75 for restarting column number one after the seventh column! 👏🏻

File: src/Jsons/Contributors.json
Comment (XanderRubio) [inline src/Jsons/Contributors.json:1]: It looks great, and thank you for reformatting the precious contributor's JSON text, as it was out of alignment.

**Review by XanderRubio — APPROVED** [review]
Hi @negar-75, 

Great job on your contribution to the website! I can relate to your desire to start your own business, generate income for yourself, create opportunities for others, and make a positive impact on the world. Thank you for taking the time to share your thoughts. By continuing to learn, create, and stay disciplined, you will achieve your goals. 

If you haven't already, please give the repository a star. I look forward to your future contributions to issue #48 and the ongoing growth of this open source project. As technology evolves, I hope this project will serve as a platform for developers to showcase their skills and demonstrate what they're capable of. Merging now! Have a great day Negar😎

File: CONTRIBUTORS.md
Comment (negar-75) [reply CONTRIBUTORS.md:1]: thanks @XanderRubio

**Comment by negar-75** [comment]
thanks @XanderRubio  for your great comment
yeah, now I am working on that and getting familiar with the structure of the project.
thanks for giving me opportunity to show myself as a junior developer.
I appreciate it

**Comment by XanderRubio** [comment]
Great, and my pleasure! One of the main values (I think we should add core values to this project now that I think about it) is to assist junior developers by giving them the opportunity to show off/improve on their current/new skills! Have a great rest of your day!

**Pull request merged**